commands/command_fsck.go

```diff
@@ -47,7 +47,7 @@ func fsckCommand(cmd *cobra.Command, args []string) {
 		}
 	})
 
-	if err := gitscanner.ScanRefWithDeleted(ref.Sha, nil); err != nil {
+	if err := gitscanner.ScanRef(ref.Sha, nil); err != nil {
```

Comment:
Could you provide a little more context why this method call was changed to no longer scan through deleted objects?

My thought was that the idea of a fsck in this context was to find and remove the LFS pointers in deleted Git objects.